scripts/polarion_sync/tests/test_polarion_sync.py

```
@@ -0,0 +1,492 @@
+from scripts.polarion_sync.polarion_client import (
+    PolarionResult,
+    _humanize_test_name,
+    clear_sibling_cache,
+    create_test_cases,
+)
+from scripts.polarion_sync.push_gate import _validate_changed_files, _validate_diff
+from scripts.polarion_sync.scanner import UnlinkedTest, scan_file
+
+
+@pytest.fixture(autouse=True)
+def _clear_polarion_caches():
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_clear_polarion_caches`

(ANN202)